docs/SUMMARY.md

```diff
@@ -7,6 +7,7 @@
   * [Usage](getting-started/usage.md)
 * [General](general/README.md)
   * [Responsive](general/responsive.md)
+  * [Ratio](general/device-pixel-ratio.md)
```

Comment:
I would call this section "Pixel Ratio" since there is also "Aspect Ratio" detailed in the Responsive section